spec/support/capybara.rb

```diff
@@ -0,0 +1,23 @@
+require 'selenium/webdriver'
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(
+    app,
+    browser: :chrome,
+    args: ['--window-size=1440,900']
+  )
+end
+
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w[headless disable-gpu] }
+  )
+
+  Capybara::Selenium::Driver.new(
+    app,
+    browser: :chrome,
+    desired_capabilities: capabilities
+  )
+end
+
+Capybara.javascript_driver = :chrome
```

Comment:
I will use Capybara.default_wait_time, because it is older version of Capybara here and I get 'undefined method' error. 

About using headless mode - yes it is recommended in normal projects, but I wanted to start with interface to show new QAs what is really happening when running specs. It is described in README how to change the driver to headless.